src/components/MemberProfileCard/memberprofile.module.css

```diff
@@ -2,7 +2,10 @@
   display: flex;
   flex-direction: column;
   padding: 10px 20px;
+  height: 360px;
   width: 30%;
+  position: sticky;
+  top: 101px;
```

Comment:
this may not work well on all screen sizes, potentially causing layout issues. To ensure responsive behavior, please implement a solution that dynamically calculates the top value based on the actual height of the navbar.